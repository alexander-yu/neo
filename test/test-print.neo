var string a = "World!";
create array<string> arr1 = {| a |};
create array<int> arr2 = {| 1, 2 |};
create array<array<int>> b = {| {|1|}, {|2|} |};
create array<matrix<int>> c = {| [[1], [2]], [[3], [4]] |};

void main() {
    print("Hello\n");
    print(a);
    print("\n");
    print(arr1);
    print("\n");
    print(arr2);
    print("\n");
    print(b);
    print("\n");
    print(c);
    print("\n");
    print(True);
}
