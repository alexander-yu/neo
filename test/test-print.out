Hello
World!
{|World!|}
{|1, 2|}
{|{|1|}, {|2|}|}
{|[[1], [2]], [[3], [4]]|}
True
